io.catenax.sealant.sealant_pass/1.0.0/SealantPass.ttl

```diff
@@ -0,0 +1,650 @@
+    bamm:see <https://commission.europa.eu/energy-climate-change-environment/standards-tools-and-labels/products-labelling-rules-and-requirements/sustainable-products/ecodesign-sustainable-products_en>;
+    bamm:dataType :DocumentEntity.
+:PerformanceCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Performance"@en;
+    bamm:description "Set of measures linked to the performance and durability of the mixtures and substances."@en;
+    bamm:dataType :PerformanceEntity.
+:ProductIdentificationCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Product identification character"@en;
+    bamm:description "Set of information to identify the product."@en;
+    bamm:dataType :ProductIdentificationEntity.
+:ApplicationCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Application characteristic"@en;
+    bamm:description "Set of information referring to the application of the substance and mixture."@en;
+    bamm:dataType :ApplicationEntity.
+:SupplierCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Supplier characteristic"@en;
+    bamm:description "Set of information to identify the supplier."@en;
+    bamm:dataType :SupplierEntity.
+:HazardsIdentification a bamm:Characteristic;
+    bamm:preferredName "Hazards Identification"@en;
+    bamm:description "Set of information to identify the hazards."@en;
+    bamm:dataType :HazardsIdentificationEntity.
+:ProductComposition a bamm:Characteristic;
+    bamm:preferredName "Product Composition"@en;
+    bamm:description "Set of information displaying the product composition."@en;
+    bamm:dataType :ProductCompositionEntity.
+:SafetyMeasureCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Safety Measure"@en;
+    bamm:description "Set of information referring to safety measures."@en;
+    bamm:dataType :SafetyMeasureEntity.
+:ProductProperties a bamm:Characteristic;
+    bamm:preferredName "Product Properties "@en;
+    bamm:description "Set of information describing the physical and chemical properties."@en;
+    bamm:dataType :ProductPropertyEntity.
+:ProductDisposal a bamm:Characteristic;
+    bamm:preferredName "Product Disposal"@en;
+    bamm:description "Set of information describin the proper disposal of the product."@en;
+    bamm:dataType :ProductDisposalEntity.
+:TransportInformation a bamm:Characteristic;
+    bamm:preferredName "Transport Information"@en;
+    bamm:description "Set of information describing the transport informations."@en;
+    bamm:dataType :TransportInformationEntity.
+:RegulatoryInformationCharacteristic a bamm:Characteristic;
+    bamm:preferredName "regulatory information characteristic"@en;
+    bamm:description "Set of information regarding the regulations."@en;
+    bamm:dataType :RegulatoryInformationEntity.
+:CarbonFootprint a bamm:Characteristic;
+    bamm:preferredName "Carbon Footprint"@en;
+    bamm:description "Set of information needed for the carbon footrpinrt ecodesign regulations."@en;
```

Comment:
footprint
ecodesign -> in the previous description you write it uppercase. Be consistent